content/translate.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How to translate content using Crowdin
+draft: false
+---
+
+If you want to start translating content for Scientific Python projects, you can use Crowdin to help you manage the translation process. You can see which projects are currently set up with Crowdin in our [GitHub project board](https://github.com/orgs/Scientific-Python-Translations/projects/1).
+
+You can also see [a similar guide in video format on YouTube]().
```

Comment:
I guess we should wait to merge this PR until after the video is completed? I take a look and provide feedback shortly.